lib/blockchains/EthereumBlockchain.js

```diff
@@ -336,5 +369,95 @@ class EthereumBlockchain {
       return { valid: false, markAs: "failed", error: err.message }
     }
   }
+
+  async sendWithdrawalCoinsTransaction(coinAddress, totalAmount, hotWallet) {
+    const txToGetFee = await this.chain.new.Transaction()
+    const chainTransaction = await this.chain.new.Transaction()
+
+    try {
+      // To calculate fee let's add tx with the full amount
+      const txValues = { toAccountName: coinAddress, amount: totalAmount.toString(), symbol: 'ether' }
+      txToGetFee.actions = [await this.chain.composeAction(chainjs.Models.ChainActionType.ValueTransfer, txValues)]
+
+      const ethFee = await txToGetFee.getSuggestedFee(chainjs.Models.TxExecutionPriority.Fast)
+      const amountToTransfer = totalAmount.minus(new BigNumber(ethFee)).minus(new BigNumber(0.0001))
```

Comment:
Why are we distracting `0.001` more than the fee from the amount?
Is the fee calculation incorrect?

Reply:
It's a hack. Because it always raise an error about insufficient amount.

Reply:
I didn't figure out how to get exact amount and just added the workaround for now.

Reply:
Hmm yeah, I think it's better to investigate that deeply, rather than distracting the magic number from transfer amount.